indico/web/templates/forms/protection_widget.html

```diff
@@ -22,6 +22,8 @@
                     $('#form-group-protected-{{ field.id }} .' + $this.val() + '-protection-message').show();
 
                     toggleAclField($('#{{ field.get_form().acl.id }}'), !isProtected);
+
+                    $('.inheriting-acl-message').trigger('protection:inheriting', [$this.val() === 'inheriting']);
```

Comment:
I think this will cause problems in cases where two protection dialogs are open at the same time (which can happen if you go through the "non-inheriting children" dialog). Make sure you only select the `.inheriting-acl-message` of this widget.